glue/core/hub.py

```diff
@@ -88,6 +89,11 @@ def subscribe(self, subscriber, message_class,
            are only passed to the subscriber if filter(message) == True.
            The default is to always pass messages.
 
+        :param priority:
+           An optional integer to set the priority of the handler. Handlers
+           are sorted such that lower integer priority handlers get called
+           first when broadcasting a message.
```

Comment:
```suggestion
           are sorted such that higher priority handlers get called first
           when broadcasting a message.
```
Only realised that looking at the test, which is running handler3-handler1-handler2.
Seems more intuitive that way, too.
`sorted` will work the same with non-integers; don't know if we should check input type nonetheless or just silently accept them.

Reply:
Oh, this is the reverse of what my intent was, I missed the `reverse=True` in the code snippet @astrofrog sent with the update `sorted` call. I would rather remove that keyword there so that the current docstring is accurate and update the test, unless you two feel strongly that reverse sorting is the way it should work.

Reply:
Thought it might have sneaked in there – but as I commented, `"priority"` to me would rather imply that the highest are done first, so maybe if you prefer ascending order, maybe it would be better to rename the keyword?

Reply:
In my head, if one things is "First priority/priority 1", that is higher priority than the thing that is "second priority/priority 2". Let's hear the tiebreaker vote from @astrofrog 🙂

Reply:
Yes, my training from the German educational and astronomical magnitude systems is leading me that way, too – for numbers ~0 - 6, for the examples coming into play here it's rather the opposite.

Reply:
In echo and glue-core, higher priority value = higher priority = do it first 😄 (we already use priorities in other places so we should remain consistent)

Reply:
Got it, consistency it is then! I committed @dhomeier's suggestions, I think this is good to go.